recipe/meta.yaml

```diff
@@ -1,55 +1,305 @@
 {% set name = "google-api-core" %}
-{% set version = "1.8.2" %}
-{% set sha256 = "16b14c0492154110d869a08be22d27440010057fd826205303c7edaa1c3cd71e" %}
+{% set version = "1.9.0" %}
+{% set sha256 = "fc1fea74bd863fb71486066e0c6b3a4dad26fb70ec61a0edcada8637feb77c68" %}
 {% set build = 0 %}
 
 package:
-  name: {{ name|lower }}
+  name: {{ name|lower }}-feedstock
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: {{ sha256 }}
 
 build:
-  number: 0
-  script: python -m pip install --no-deps --ignore-installed .
-
-requirements:
-  build:
-    - python
-    - setuptools
-    - pip
-
-  run:
-    - python
-    - googleapis-common-protos >=1.5.3,<2.0dev
-    - protobuf >=3.4.0
-    - google-auth >=0.4.0,<2.0.0dev
-    - requests >=2.18.0,<3.0.0dev
-    - setuptools >=34.0.0
-    - six >=1.10.0
-    - pytz
-    # Backport of concurrent.futures for Python < 3.2.
-    - futures >=3.0.0  # [py2k]
-
-test:
-  imports:
-    - google.api_core.exceptions
+  number: {{ build }}
+
+outputs:
+  - name: {{ name }}
+    build:
+      script: python -m pip install . --no-deps --ignore-installed --verbose
+    requirements:
+      host:
+        - python
+        - pip >=18.1
+        - futures >=3.2.0  # [py<32]
+        - google-auth >=0.4.0
+        - googleapis-common-protos >=1.5.3
+        - pytz >=2019.1
+        - requests >=2.18.0
+        - setuptools >=34.0.0
+        - six >=1.10.0
+      run:
+        - {{ pin_compatible('futures') }}  # [py<32]
```

Comment:
Nice, thank you for adding `pin_compatible`. I was always a bit nervous about major semver changes.